librosa/core/audio.py

```diff
@@ -265,17 +265,17 @@ def __audioread_load(path, offset, duration, dtype: DTypeLike):
 
             if s_end < n:
                 # the end is in this frame.  crop.
-                frame = frame[: s_end - n_prev]
+                frame = frame[: int(s_end - n_prev)]  # pragma: no cover
```

Comment:
no cover here because we're deprecating this function anyway, not particularly worried about it